src/popover/src/Popover.js

```diff
@@ -237,12 +238,16 @@ export default class Popover extends Component {
 
   renderTarget = ({ getRef, isShown }) => {
     const { children } = this.props
+    const isTooltipInside = children.type === Tooltip
```

Comment:
Is it possible to _not_ pass `children`? If so we may want to check that it exists first.